src/main/java/Entities/Customer.java

```diff
@@ -0,0 +1,65 @@
+package Entities;
+

```

Comment:
remove all the extra lines

Reply:
Thankyou so much for your valuable comments, I've made all the changes..